config/defaults.mk

```diff
@@ -552,6 +552,9 @@ ALGOIM_OPT = -I$(ALGOIM_DIR)/src $(BLITZ_OPT)
 ALGOIM_LIB = $(BLITZ_LIB)
 
 # BENCHMARK library configuration
+ifeq ($(MFEM_USE_BENCHMARK),YES)
+   BASE_FLAGS = -std=c++14
+endif
```

Comment:
Note that this will have to be modified due to [this change](https://github.com/mfem/mfem/pull/4452/commits/4606b273d8ece2a33233cb40ff6a4f79a4d84656) in https://github.com/mfem/mfem/pull/4452